Groovy/Groovy.sublime-syntax

```diff
@@ -173,34 +178,8 @@ contexts:
     - include: keyword-language
 
   keyword-language:
-    - match: \bthrow\b
-      scope: keyword.control.flow.throw.groovy
-    - match: \btry\b
-      scope: keyword.control.exception.try.groovy
-    - match: \bfinally\b
-      scope: keyword.control.exception.finally.groovy
-    - match: \bcatch\b
-      scope: keyword.control.exception.catch.groovy
-    - match: \bdefault\b
+    - match: \b(?:default|else\s+if|else|if|switch)\b
       scope: keyword.control.conditional.groovy
-    - match: \bfor\b
-      scope: keyword.control.loop.for.groovy
-    - match: \bwhile\b
-      scope: keyword.control.loop.while.groovy
-    - match: \bdo\b
-      scope: keyword.control.loop.do-while.groovy
-    - match: \breturn\b
-      scope: keyword.control.flow.return.groovy
-    - match: \bbreak\b
-      scope: keyword.control.flow.break.groovy
-    - match: \bcontinue\b
-      scope: keyword.control.flow.continue.groovy
-    - match: \bif\b
-      scope: keyword.control.conditional.if.groovy
-    - match: \belse\b
-      scope: keyword.control.conditional.else.groovy
-    - match: \bswitch\b
-      scope: keyword.control.conditional.switch.groovy
```

Comment:
Why this step backwards? Nearly all "recently" refactored default syntaxes use this level of detail.

Reply:
Indeed, these were only recently fixed (in https://github.com/sublimehq/Packages/pull/4051) - it would be a shame to regress them already lol